wagtailio/settings/base.py

```diff
@@ -311,6 +311,9 @@
 if "SERVER_EMAIL" in env:
     SERVER_EMAIL = DEFAULT_FROM_EMAIL = env["SERVER_EMAIL"]
 
+# Notification emails
+
+WAGTAILADMIN_NOTIFICATION_INCLUDE_SUPERUSERS = False
```

Comment:
suggestion: make the value an env variable like the rest. this way we can toggle on/off when needed without a new PR/deployment. e.g.

```suggestion
WAGTAILADMIN_NOTIFICATION_INCLUDE_SUPERUSERS = env.get("MODERATION_NOTIFY_SUPERUSERS", False)
```

Reply:
Works for me @zerolab. Great idea.